Makefile

```diff
@@ -0,0 +1,25 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+TARGET ?= skbio
+
+ifeq ($(WITH_COVERAGE), TRUE)
+	COVERAGE = --with-coverage
+endif
+
+ifeq ($(WITH_DOCTEST), FALSE)
+	DOCTEST =
+else
+	DOCTEST = --with-doctest
+endif
+
+test:
+	nosetests -v $(TARGET) $(DOCTEST) $(COVERAGE) --ignore DO_NOT_IGNORE_ANYTHING
```

Comment:
Sure, I originally added it as a lazy way to keep [this comment](https://github.com/biocore/scikit-bio/blame/30357156ec94e74012d653cb566eeff6c3b9af42/doc/source/development/coding_guidelines.rst#L272) accurate. We could add a ``VERBOSE=TRUE`` option to the makefile or edit the comment to not include an example.